src/main/kotlin/it/pagopa/wallet/eventdispatcher/service/WalletCDCService.kt

```diff
@@ -0,0 +1,26 @@
+package it.pagopa.wallet.eventdispatcher.service
+
+import it.pagopa.wallet.eventdispatcher.audit.LoggingEvent
+import java.util.*
+import org.slf4j.LoggerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+
+@Component
+class WalletCDCService(
+    private val cdcKafkaTemplate: KafkaTemplate<String, Any>,
+    private val cdcTopicName: String
+) {
+
+    private val log = LoggerFactory.getLogger(WalletCDCService::class.java.name)
+
+    fun sendToKafka(event: LoggingEvent): Mono<Unit> {
+        log.info("Sending CDC event to Kafka: [{}]", event.id)
+
+        return Mono.fromCallable { cdcKafkaTemplate.send(cdcTopicName, event.id, event) }
```

Comment:
why not use [ReactiveKafkaProducerTemplate](https://docs.spring.io/spring-kafka/api/org/springframework/kafka/core/reactive/ReactiveKafkaProducerTemplate.html) directly?

Reply:
Looking at the above comment this Mono.fromCallable api call should be replaced here since ReactiveKafkaProducerTemplate will return a Mono. Right?

Reply:
yes i was just noticing the same thing,
solved here 3dfceb07920030f93b4e171fa32744b9a78ae165